packages/components/src/Textarea/Textarea.tsx

```diff
@@ -1,22 +1,58 @@
 import React from 'react';
-import { useStyles, system } from '@marigold/system';
+import { useStyles } from '@marigold/system';
+import { Exclamation, Required } from '@marigold/icons';
+import { ComponentProps } from '@marigold/types';
 
-export type TextareaProps = {
+import { ValidationMessage } from '../ValidationMessage';
+import { Label } from '../Label';
+import { Box } from '../Box';
+
+type TextareaProps = {
+  // how to check if theres a label and then htmlFor is required
   variant?: string;
-};
+  label?: string;
+  htmlFor?: string;
+  error?: string;
```

Comment:
I would expect error to be boolean, maybe rename it to errorMessage

Reply:
Made it like in field Component. We should use both in the same way but yes maybe errorMessage is better

Reply:
Then I would consider changing Field aswell so it's clear :)